.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Why this change to JDK 1_7?

Reply:
Android studio doing sh*t

Reply:
Happens to me too, despite having and using jdk 8

Reply:
ah ok, don't worry

Reply:
https://developer.android.com/studio/write/java8-support
You can give a try to this, worked for me.

Reply:
This changes the jdk at module level, while the file misc is at project level. Doesn' t work

Reply:
My bad it's the project's misc.xml file, didn't notice

Reply:
Ok it worked now, no clue why😂

Reply:
Wait what the heck

Reply:
Dunno, now misc.xml is not showing up because it's not different from its version in develop branch.